R/scalar_in_linter.R

```diff
@@ -1,12 +1,13 @@
 #' Block usage like x %in% "a"
 #'
 #' `vector %in% set` is appropriate for matching a vector to a set, but if
-#'   that set has size 1, `==` is more appropriate. `%chin%` from `{data.table}`
-#'   is matched as well.
+#'   that set has size 1, `==` is more appropriate.
 #'
 #' `scalar %in% vector` is OK, because the alternative (`any(vector == scalar)`)
 #'   is more circuitous & potentially less clear.
 #'
+#' @param in_operators Character vector of additional functions that behave like the `%in%` operator
```

Comment:
```suggestion
#' @param in_operators Character vector of additional infix operators that behave like the `%in%` operator
```

"function" might imply any `foo(.)` can be linted, but the XPath logic is very specific to `//SPECIAL`, so we should emphasize that.

Reply:
Suggestion added